Source/WebCore/platform/graphics/transforms/RotateTransformOperation.h

```diff
@@ -73,6 +73,16 @@ class RotateTransformOperation final : public TransformOperation {
         return false;
     }
 
+    bool applyUnrounded(TransformationMatrix& transform, const FloatSize& /*borderBoxSize*/) const override
+    {
+        if (type() == TransformOperation::Type::Rotate)
+            transform.rotate(m_angle, false);
```

Comment:
Boolean trap here. Better to add an enum for the `snap90degRotations` argument.

Reply:
Called it in https://github.com/WebKit/WebKit/pull/19462#issuecomment-1776697276.